pkg/manager/member/tikv_scaler.go

```diff
@@ -142,24 +144,24 @@ func (s *tikvScaler) ScaleIn(meta metav1.Object, oldSet *apps.StatefulSet, newSe
 			// TODO: double check if store is really not in Up/Offline/Down state
 			klog.Infof("TiKV %s/%s store %d becomes tombstone", ns, podName, id)
 
-			pvcName := ordinalPVCName(v1alpha1.TiKVMemberType, setName, ordinal)
-			pvc, err := s.deps.PVCLister.PersistentVolumeClaims(ns).Get(pvcName)
+			pvcs, err := util.ResolvePVCFromPod(pod, s.deps.PVCLister)
+			klog.V(4).Infof("ResolvePVCFromPod: %+v", pvcs)
 			if err != nil {
-				return fmt.Errorf("tikvScaler.ScaleIn: failed to get pvc %s for cluster %s/%s, error: %s", pvcName, ns, tcName, err)
-			}
-			if pvc.Annotations == nil {
-				pvc.Annotations = map[string]string{}
+				return fmt.Errorf("tikvScaler.ScaleIn: failed to get pvcs for pod %s/%s in tc %s/%s, error: %s", ns, pod.Name, ns, tcName, err)
 			}
-			now := time.Now().Format(time.RFC3339)
-			pvc.Annotations[label.AnnPVCDeferDeleting] = now
-			_, err = s.deps.PVCControl.UpdatePVC(tc, pvc)
-			if err != nil {
-				klog.Errorf("tikv scale in: failed to set pvc %s/%s annotation: %s to %s",
-					ns, pvcName, label.AnnPVCDeferDeleting, now)
-				return err
+			for _, pvc := range pvcs {
```

Comment:
Suggest updating the e2e cases to cover the code change in this file.

Reply:
will add e2e case in another PR

Reply:
ok